.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for concrete
+title: '[FeatureReq]'
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
(not needing any change, just a suggestion)
I see your point, what do you think of **What is the problem you want to solve and can not with the current version**